Tests/SonarScanner.MSBuild.PreProcessor.Test/ArgumentProcessorTests.cs

```diff
@@ -217,7 +217,25 @@ public void PreArgProc_Region_Invalid()
         @"The arguments 'sonar.host.url' and 'sonar.scanner.sonarcloudUrl' are both set. Please set only 'sonar.scanner.sonarcloudUrl'.")]
     [DataRow(null, "https://host", null, "https://api", typeof(ServerHostInfo), "https://host", "https://api")]
     [DataRow(null, "https://host", null, null, typeof(ServerHostInfo), "https://host", "https://host/api/v2")]
-    public void PreArgProc_Region_Overrides(string region, string hostOverride, string sonarClourUrlOverride, string apiOverride, Type expectedHostInforType, string expectedHostUri, string expectedApiUri, params string[] expectedWarnings)
+
```

Comment:
The DataRow block should be continues. If you need to introduce some new big section, please put a comment here instead
```suggestion
```